modules/git/commit.go

```diff
@@ -177,13 +190,18 @@ func CommitsCountFiles(ctx context.Context, repoPath string, revision, relpath [
 }
 
 // CommitsCount returns number of total commits of until given revision.
-func CommitsCount(ctx context.Context, repoPath string, revision ...string) (int64, error) {
-	return CommitsCountFiles(ctx, repoPath, revision, []string{})
+func CommitsCount(ctx context.Context, opts CommitsCountOptions) (int64, error) {
+	return CommitsCountFiles(ctx, opts)
 }
 
 // CommitsCount returns number of total commits of until current revision.
-func (c *Commit) CommitsCount() (int64, error) {
-	return CommitsCount(c.repo.Ctx, c.repo.Path, c.ID.String())
+func (c *Commit) CommitsCount(not string) (int64, error) {
```

Comment:
There is only one call to `commit.CommitsCount(not)`

I think it better to keep the old `commit.CommitsCount()` (no `not` argument)

And make that only-one `commit.CommitsCount(not)` to use `CommitsCountOptions`. Then a lot of code deosn't need to change.